.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,26 @@
 Closes #
 
-<!-- 
-Thank you for contributing to ODK Collect!
+## Description
 
-Before sending this PR, please read
-https://github.com/getodk/collect/blob/master/CONTRIBUTING.md
--->
+Please provide a brief description of the changes in this PR.
 
-#### What has been done to verify that this works as intended?
+### Changes
 
-#### Why is this the best possible solution? Were any other approaches considered?
+List the changes done to fix a bug or introduce a new feature.
 
-#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
+## Checklist
 
-#### Do we need any specific form for testing your changes? If so, please attach one.
+Please check the boxes below that apply:
 
-#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
```

Comment:
This statement is not required, the checklist below should suffice.

Reply:
The line was there in the original PR Template. I deleted it in here. The original template dosen't contain this. I have attached the link to the original one below.

https://github.com/getodk/collect/blob/9c6b34833c2c9b2ff3cc063e39bb110ea0e16663/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L18